ape_infura/providers.py

```diff
@@ -8,7 +8,24 @@
 from web3.gas_strategies.rpc import rpc_gas_price_strategy
 from web3.middleware import geth_poa_middleware
 
-_ENVIRONMENT_VARIABLE_NAMES = ("WEB3_INFURA_PROJECT_ID", "WEB3_INFURA_API_KEY")
+_ENVIRONMENT_VARIABLE_OPTIONS = {
+    "ethereum": (
+        "WEB3_INFURA_PROJECT_ID",
+        "WEB3_INFURA_API_KEY",
+    ),
+    "arbitrum": (
+        "WEB3_ARBITRUM_INFURA_PROJECT_ID",
+        "WEB3_ARBITRUM_INFURA_API_KEY",
+    ),
+    "optimism": (
+        "WEB3_OPTIMISM_INFURA_PROJECT_ID",
+        "WEB3_OPTIMISM_INFURA_API_KEY",
+    ),
+    "polygon": (
+        "WEB3_POLYGON_INFURA_PROJECT_ID",
+        "WEB3_POLYGON_INFURA_API_KEY",
+    ),
+}
```

Comment:
Just noticed this. You actually don't need this change because Infura uses just one project ID for all of it's RPC endpoints, unlike Alchemy which uses one per network

Reply:
thanks for clarifying.
fixed the issue. tests work locally for me, so shall be good in CI as well